bundles/AdminBundle/Controller/Admin/DataObject/DataObjectController.php

```diff
@@ -1073,78 +1092,13 @@ public function updateAction(Request $request)
         return $this->adminJson(['success' => $success]);
     }
 
-    /**
-     * @param DataObject\AbstractObject $updatedObject
-     * @param int $newIndex
-     */
-    protected function updateIndexesOfObjectSiblings(DataObject\AbstractObject $updatedObject, $newIndex)
-    {
+    private function executeInsideTransaction(callable $fn) {
```

Comment:
I would move this to model

Reply:
Not directly related, but I would do this in one go. There is even a task for this 
https://github.com/pimcore/pimcore/issues/6906